### ISSUE TITLE
1.0.0 Release

### ISSUE THREAD

**Issue by nblumhardt** [bytenik/Seq.App.Slack#62]
Unfortunately, NuGet.org seems to think that 0.* versions are "newer" than 1.0-* versions, making the latest version harder to determine:

https://www.nuget.org/packages/seq.app.slack

This package is now one of the most battle-tested Seq apps in use, it seems like a good time to mark it 1.0. I'm happy sending through the version update, what do you think, @bytenik?

**Comment by bytenik** [comment]
I agree. That's also a little bit absurd that NuGet is ignoring semver like that.
